CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java

```diff
@@ -558,7 +610,7 @@ public void setExpirationError_setsError() {
 
     @Test
     public void setExpirationError_doesNotRequestFocusIfCardNumberIsAlreadyFocused() {
```

Comment:
All of the error focus tests probably need to be update to take into account the cardholder name.